apps/builder/lib/constants.ts

```diff
@@ -52,7 +52,7 @@ export const NETWORK_OPTIONS = [
   { namespace: 'eip155', network: zksync },
   { namespace: 'eip155', network: base },
   { namespace: 'solana', network: solana },
-  { namespace: 'solana', network: solanaTestnet },
+  { namespace: 'solana', network: solanaDevnet },
```

Comment:
why this change?

Reply:
At network list section on Demo we've been been using devnet id, while here it's Testnet, just aligned them